packages/configuration-screen/src/setupConfiguration.ts

```diff
@@ -0,0 +1,141 @@
+import { updateConfigurationFile } from './configurationFileHelper';
+import { Tenant } from './models/flare';
+import {
+    SplunkApplicationNamespace,
+    SplunkService,
+    SplunkStoragePasswordAccessors,
+} from './models/splunk';
+import { promisify } from './util';
+
+const appName: string = 'flare';
+const storageRealm: string = 'flare_integration_realm';
+const applicationNameSpace: SplunkApplicationNamespace = {
+    owner: 'nobody',
+    app: appName,
+    sharing: 'app',
+};
+
+async function completeSetup(splunkService: SplunkService): Promise<void> {
+    var configurationFileName = 'app';
+    var stanzaName = 'install';
+    var propertiesToUpdate = {
+        is_configured: 'true',
+    };
+
+    await updateConfigurationFile(
+        splunkService,
+        configurationFileName,
+        stanzaName,
+        propertiesToUpdate
+    );
+}
+
+async function reloadApp(splunkService: SplunkService, appName: string): Promise<void> {
+    var splunkApps = splunkService.apps();
+    await promisify(splunkApps.fetch)();
+
+    var currentApp = splunkApps.item(appName);
+    await promisify(currentApp.reload)();
+}
+
+function redirectToHomepage(appName: string): void {
+    var redirectUrl = '/app/' + appName;
+    window.location.href = redirectUrl;
+}
+
+function createService(applicationNamespace: SplunkApplicationNamespace): SplunkService {
+    var http = new splunkjs.SplunkWebHttp();
+    var service = new splunkjs.Service(http, applicationNamespace);
+
+    return service;
+}
+
+function retrieveUserTenants(
+    serverKey: string,
+    successCallback: (userTenants: Array<Tenant>) => void,
+    errorCallback: (errorMessage: string) => void
+): void {
+    const service = createService(applicationNameSpace);
+    const data = { serverKey: serverKey };
```

Comment:
Can't remember what we said. Are we changing these in this MR or get this MR merged first?

Reply:
Those changes will be part of this PR: https://github.com/Flared/splunk-integration/pull/16, I'm waiting for this PR to be done to push my local changes